src/lib.rs

```diff
@@ -263,7 +263,7 @@ impl Message {
     /// # Examples
     ///
     /// ```
-    /// # #[cfg(feature = "bitcoin-hashes")] {
+    /// # #[cfg(feature = "bitcoin_hashes")] {
```

Comment:
Since this "from the outside" of the library I left this one as "bitcoin-hashes"

Reply:
I considered this, but decided that since it was hidden it counts as "in the library".